tests/test_other.py

```diff
@@ -3789,6 +3789,12 @@ def test_init_file_at_offset(self):
     Popen([PYTHON, EMCC, 'src.cpp']).communicate()
     self.assertContained('read: 0\nfile size is 104\n', run_js('a.out.js'))
 
+  def test_unlink(self):
+    self.emcc_args = []
+    src = path_from_root('tests', 'fs', 'test_unlink.c')
+    out = path_from_root('tests', 'fs', 'test_unlink.out')
+    self.do_run_from_file(src, out, js_engines=[NODE_JS])
+
```

Comment:
Ah, sorry, how we do tests is somewhat confusing. `self.emcc_args` and `do_run_from_file` are just for the `core` test suite.

I opened #6473 now to add a simple helper function for running tests in the `other` suite, with an example there which is how the test here could be done.

Otherwise, lgtm, thanks!